acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/AbstractCmsPage.java

```diff
@@ -0,0 +1,25 @@
+package uk.nhs.digital.ps.test.acceptance.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.interactions.Actions;
+import uk.nhs.digital.ps.test.acceptance.webdriver.WebDriverProvider;
+
+public class AbstractCmsPage extends AbstractPage {
+
+    static protected final String URL = "http://localhost:8080/cms";
+
+    AbstractCmsPage(final WebDriverProvider webDriverProvider) {
+        super(webDriverProvider);
+    }
+
+    public void logout() {
+
+        // @formatter:off
```

Comment:
Just wondering why the formatter:off is needed? Seems to format OK in IntelliJ for me.

Reply:
Force of habit - we used auto-formatting a lot on my previous project and we'd often exclude this type of expressions from its scope like this, to preserve readability.

Since we don't use auto-formatting here, there is no need for these `// @formatter:off`/`// @formatter:on` statements - I have now removed them.